schemas/com.iterable/system_webhook/jsonschema/1-0-0

```diff
@@ -0,0 +1,695 @@
+        "city": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "City (e.g., 'San Jose')",
+          "maxLength": 4096
+        },
+        "contentAvailable": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Is content available"
+        },
+        "contentId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the content the event is associated with (e.g., 3681)"
+        },
+        "country": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Country (e.g., 'United States')",
+          "maxLength": 4096
+        },
+        "createdAt": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Creation timestamp (e.g., '2017-05-15 23:59:47 +00:00')",
+          "maxLength": 255
+        },
+        "deeplink": {
+          "type": [
+            "object",
+            "null"
+          ],
+          "properties": {
+            "android": {
+              "type": [
+                "string",
+                "null"
+              ],
+              "maxLength": 4096
+            },
+            "ios": {
+              "type": [
+                "string",
+                "null"
+              ],
+              "maxLength": 4096
+            }
+          },
+          "additionalProperties": true,
+          "description": "Deep Link. A mapping that accepts two optional properties: 'ios' & 'android' and their respective deep link values."
+        },
+        "device": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Device application name (e.g., 'Gmail')",
+          "maxLength": 4096
+        },
+        "email": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Reference to the unique user the event was applied to.",
+          "maxLength": 320,
+          "format": "email"
+        },
+        "emailId": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Reference to the email associated with the event (e.g., 'c59667:t93849:docs@iterable.com')",
+          "maxLength": 4096
+        },
+        "emailIds": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": "string",
+            "maxLength": 4096,
+            "description": "Reference to email"
+          },
+          "description": "References to emails associated with the event"
+        },
+        "emailListIds": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": [
+              "integer",
+              "null"
```

Comment:
Same question as above, about nulls in the array.

Reply:
Good point, changed it not to allow nulls inside the list.